.storybook/decorators/index.js

```diff
@@ -123,20 +182,83 @@ export const withSizingWrapper = makeDecorator({
 			disable: true,
 		};
 
-		return html` <div class="spectrum-Examples">
-			${sizes.map((size) => {
-				context.args.size = size;
-				return html` <div class="spectrum-Examples-item" data-value=${size}>
-					<div class="spectrum-Examples-itemGroup" id="scoped-root">
-						${StoryFn(context)}
-					</div>
-					<h4
-						class="spectrum-Detail spectrum-Detail--sizeXS spectrum-Examples-itemHeading"
-					>
-						${printSize(size)}
-					</h4>
-				</div>`;
-			})}
-		</div>`;
+		return isChromatic()
+			? html` <div class="spectrum-Examples">
+					${sizes.map((size) => {
+						context.args.size = size;
+						return html` <div class="spectrum-Examples-item" data-value=${size}>
+								<h4
+									class="spectrum-Detail spectrum-Detail--sizeXS spectrum-Examples-itemHeading"
+								>
+									${printSize(size)}
+								</h4>
+							<div class="spectrum-Examples-itemGroup" id="scoped-root">
+								${StoryFn(context)}
+							</div>
+						</div>`;
+					})}
+				</div>`
+			: StoryFn(context);
 	},
 });
+
+export const withBackgroundColorDisplayWrapper = makeDecorator({
+  name: "withBackgroundColorDisplayWrapper",
+  parameterName: "context",
+  wrapper: (StoryFn, context) => {
+    const { globals, args, parameters } = context;
+    const defaultBackgroundColor = 'default';
+    const backgroundColorDisplay = parameters.backgroundColorDisplay || globals.backgroundColorDisplay || defaultBackgroundColor;
+    const isDefault = backgroundColorDisplay === defaultBackgroundColor;
+    const isStacked = backgroundColorDisplay === 'stacked';
+    const bodyEl = document.body;
+    const bodyElClasses = bodyEl.classList;
+
+    // remove the classes from the Storybook body element
+    // use a timeout here to make it wait for other addons to modify classes first and then remove them after that
+    useEffect(() => {
+      if (isDefault) {
+        setTimeout(() => {
+          bodyEl.className = bodyElClasses;
+        }, 1000);
+      } else {
+        setTimeout(() => {
+          bodyEl.classList.remove(...backgroundColorClassNames);
+        }, 1000);
+      }
+    }, [bodyEl]);
+
+    // is it the "default" display option? Return early
+    if (isDefault) return StoryFn(context);
+
+    // prep the display areas with the correct classes
+    const isExpress = args.express;
+
+    const backgroundColorClassNames = ['spectrum--light', 'spectrum--dark', 'spectrum--darkest'];
```

Comment:
will including darkest cause issues when we move to s2 and no longer support darkest?

Reply:
Good callout! I can probably make this dynamic.